app/views/mailers/agent_notifications/conversation_notifications_mailer/conversation_creation.html.erb

```diff
@@ -1,4 +1,5 @@
-<p>Hi <%= @agent.name %>,</p>
+<p>Hi {{agent.name}}</p>
```

Comment:
Do all these files always rendered using Liquid templating engine and is yes would the extension still be `.html.erb` or something like `.liquid.erb`. make more sense?

Reply:
added a view handler for liquid.